submissions/kissik/data-structures/index.js

```diff
@@ -0,0 +1,110 @@
+const http = require('http');
+
+const fs = require('fs');
+
+const port = process.env.PORT || 3210;
+
+const list = {};
+
+list.data = {};
+
+list.push = data => {
+  list.data = {
+    item: data,
+    next: list.data
+  };
+};
+
+list.pop = () => {
+  const a = list.data.item ? list.data.item : 'none';
```

Comment:
Please, don't give such names to variables because they do not descriptive.

Reply:
done: a -> itemValue